portfolio/src/main/webapp/script.js

```diff
@@ -1,17 +1,8 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
+/** 
+ * Validates the user's inputted number of comments to be displayed, called on input.
+ * @return {boolean} Whether user input was a valid, postive integer.
```

Comment:
nit: positive

Reply:
fixed! Nice catch, someone should make a spell-check plugin for IDEs.